### PR TITLE
Use develop for comparing lint rules

### PR DESCRIPTION
**Pull request by pablopunk**

### CODE REVIEW

**Comment by petrjasek** [comment]
seems like it doesn't get develop branch on travis, I guess running `git fetch develop` would be good, could be part of `.travis.yml`

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ cache:
         - $HOME/.cache/yarn
 
 script:
+    - git fetch origin
```
Comment (petrjasek) [inline .travis.yml]: maybe `git branch develop origin/develop` will do the trick

Comment (petrjasek) [reply .travis.yml]: but you have to probably do that fetch first

File: scripts/core/editor3/components/Editor3Component.tsx [hunk 2]
```diff
@@ -212,7 +212,7 @@ export class Editor3Component extends React.Component<IProps> {
         return !canDropMedia(e, this.props);
     }
 
-    handleDropOnEditor(selection: SelectionState, dataTransfer: Object, isInternal: DraftDragType): DraftHandleValue {
+    handleDropOnEditor(selection: SelectionState, dataTransfer: any, isInternal: DraftDragType): DraftHandleValue {
```
Comment (tomaskikutis) [inline scripts/core/editor3/components/Editor3Component.tsx]: `dataTransfer` argument is not used at all. Can you remove it?

Comment (pablopunk) [reply scripts/core/editor3/components/Editor3Component.tsx]: @tomaskikutis the argument is coming from https://draftjs.org/docs/api-reference-editor.html#handledrop

**Pull request merged**